test/queryClient.test.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020-2022 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { GraphqlQueryClient } from '../packages/network-clients';
+import { NETWORK_CONFIGS } from '../packages/network-clients/src/config';
+import assert from 'assert';
+import { GET_INDEXER } from '../packages/network-clients/src/graphql/indexers';
+
+describe('query client', () => {
+  let client: GraphqlQueryClient;
+
+  beforeAll(async () => {
+    const config = NETWORK_CONFIGS.kepler;
+    assert(config, 'network config not defined');
+    client = new GraphqlQueryClient(config);
+  }, 160000);
+
+  it('can query indexer detail', async () => {
+    const apolloClient = client.explorerClient;
+    const result = await apolloClient.query({
```

Comment:
Try to destruct the `result`, and check each of them, the value can not be undefined:
```
const { 
   id,
    metadata,
    controller,
    commission,
    totalStake
}  = result;
```